### PR TITLE
Add "new users" label to GitGitGadget PRs of first-time users

### PR DESCRIPTION
**Pull request by dscho**
This was suggested in https://github.com/gitgitgadget/gitgitgadget/issues/414#issuecomment-762022194 (among other good suggestions).

### CODE REVIEW

**Review by derrickstolee — APPROVED** [review]

File: tsconfig.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     /* Basic Options */
-    "target": "es6",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+    "target": "es2018",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
```
Comment (webstech) [inline tsconfig.json:4]: Should lib be updated in synch with target?  Current recommendations for node 12 are [in this repo](https://github.com/tsconfig/bases/tree/master/bases).  fyi, have been running with latest node without issues so we may want to consider updating the node version in package.json.

Comment (dscho) [reply tsconfig.json]: > 
> 
> Should lib be updated in synch with target? Current recommendations for node 12 are [in this repo](https://github.com/tsconfig/bases/tree/master/bases). fyi, have been running with latest node without issues so we may want to consider updating the node version in package.json.

Makes sense! Pushed.

File: tests/ci-helper.test.ts [hunk 2]
```diff
@@ -61,6 +62,12 @@ class TestCIHelper extends CIHelper {
         this.ghGlue.updatePR = updatePR;
         this.updatePRCalls = updatePR.mock.calls;
 
+        const addPRLabels = jest.fn( async (_:string, labels: string[]):
```
Comment (webstech) [inline tests/ci-helper.test.ts:65]: I guess there is no linting rule to require a space here `_:string`.  I will let you decide if you want to update it.

**Review by webstech — DISMISSED** [review]
Thanks for getting this done.  I will let you decide on the space before merging.

File: tests/ci-helper.test.ts (hunk 2 above)
Comment (dscho) [reply tests/ci-helper.test.ts]: Good point. Fixed.

**Review by webstech — APPROVED** [review]
Thanks.

**Pull request merged**